### DISCUSSION TITLE
Vanilla preset vs Enhanced edition preset?

### DISCUSSION THREAD

**Discussion by PinkiePieFTW** [Aelto/W3_RandomEncounters_Tweaks#208] [Q&A]
Hi
What's the difference beetwen Vanilla preset and Enhanced edition preset?

**Comment by Aelto** [comment]
Hello,

The EE preset tweaks of few mod settings to make it easier for the player, it takes in consideration that in EE you must prepare before a fight so ambushes are rarer and that most vampires are really strong so their chances to spawn are lowered. It also goes from the "Random" difficulty to the "normal" or "hard" to limit the size of the groups you see.

The Vanilla preset is what you get when you use the `default` preset in the menus.

_(i've deleted your comment from nexusmods to avoid duplicates)_

**Comment by PinkiePieFTW** [comment]
Very much appreciated! Great mod.
Thank you :)
